### ISSUE TITLE
Default Printer properties appear to be hard coded.

### ISSUE THREAD

**Issue by kd7vea** [Libki/libki-manual#5]
we are in the process of setting up Libki for our organization and had some trouble when we got to the printer section.  We would run the Enable google cloud print script, but would always get a 404 error message from the link produced.  We realized that the example template given in the install manual was hardcoded in  /home/libki/libki-server/libki_local.conf.  once we went in and modified the file with the correct ID, Secret, and printer IDs, everything worked.  I had reinstalled a second time trying to get around the 404 error with no luck, so our in-house Programmer installed on a VM with the same issue.  he dug into the code and found the issue, so if it's not hardcoded, there is something else that is dumping the default info there.  This looks like it's going to be a great system and we are excited to switch out the old system

**Comment by kylemhall** [comment]
Can you please describe the change you made to fix your issue?

**Comment by kd7vea** [comment]
To fix the Issue, I Navigated to the libki_local.conf file at /home/libki/libki-server/libki_local.conf.  in the <Printer> section we noticed that the Client_id, client_secret, and google_cloud_id for the printer were all set to the ID's from the example in the manual.  We had already set that information in the Print management form at <server-IP>:3000/administration/settings in the Print Management tab, and were sure to click on Update.  At this point we took the three ID variables from that form and changed them in libki_local.conf.  so it appears that the example YAML format information from the manual is pre-set in the libki_local.conf file, and trying to update from the web interface is not updating the file.
